corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -157,29 +161,17 @@ hqDefine("linked_domain/js/domain_links", [
         self.linked_domain = ko.observable(link.linked_domain);
         self.is_remote = link.is_remote;
         self.master_domain = link.master_domain;
-        self.remote_base_url = ko.observable(link.remote_base_url);
         self.lastUpdate = link.last_update;
-        if (self.is_remote) {
-            self.domain_link = self.linked_domain;
-        } else {
-            self.domain_link = initialPageData.reverse('domain_links', self.linked_domain());
-        }
+        self.upstreamURL = link.upstream_url;
```

Comment:
this is mostly a nit but for acronyms, the best practice is to only capitalize the first character. eg `upstreamUrl`

Reply:
Thanks for the tip 👍